src/lib/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { type MouseEventHandler, useEffect, useState } from 'react'
+
+import { InfoIcon } from 'lib/icons/Info.js'
+import { InfoDarkIcon } from 'lib/icons/InfoDark.js'
+
+interface TooltipProps {
+  children: React.ReactNode
```

Comment:
```suggestion
  children: JSX.Element
```

Reply:
This will make TS complain about strings. I'll use `PropsWithChildren` from `react` instead unless there are any objections.

Reply:
Ah yea my bad. Should have recommended that instead. If you search the existing code base there should be a lot of examples on how we currently use PropsWithChildren